contracts/vaults/Claim.sol

```diff
@@ -24,11 +27,14 @@ contract Claim is Base {
     }
 
     /**
-    * @notice Called by a committee to submit a claim for a bounty.
-    * The submitted claim needs to be approved or dismissed by the Hats governance.
-    * This function should be called only on a safety period, where withdrawals are disabled.
-    * Upon a call to this function by the committee the vault's withdrawals will be disabled
-    * until the Hats governance will approve or dismiss this claim.
+    * @notice Called by the committee to submit a claim for a bounty payout.
+    * The submitted claim needs to be approved or dismissed by governance.
```

Comment:
This is not correct, the governance has no role anymore in approve of dismiss. Only the arbitrator can challenge approve or dismiss, or anyone can approve or dismiss after certain timeouts.